components/app.js

```diff
@@ -0,0 +1,29 @@
+class App {
+  handleGetGradesError(error) {
+    console.error(error);
+  }
+  handleGetGradesSuccess(grades) {
+    this.gradeTable.updateGrades(grades);
+  }
+  constructor(gradeTable) {
+    this.handleGetGradesError = this.handleGetGradesError.bind(this),
+    this. handleGetGradesSuccess = this.handleGetGradesSuccess.bind(this),
+    this.gradeTable = gradeTable
+  }
```

Comment:
Move the constructor to the top, make it the first thing in the class pls

Reply:
done